src/component/books/AddBooks.js

```diff
@@ -20,21 +20,23 @@ const AddBooks = () => {
     setCategory('');
   };
   return (
-    <form onSubmit={submitBookToStore}>
+    <form onSubmit={submitBookToStore} className="form">
       <h2>Add new Books</h2>
-      <input
-        type="text"
-        placeholder="book title"
-        onChange={(e) => setTitle(e.target.value)}
-        value={title}
-      />
-      <input
-        type="text"
-        placeholder="category"
-        onChange={(e) => setCategory(e.target.value)}
-        value={category}
-      />
-      <button type="submit">submit</button>
+      <div className="form-inputs">
+        <input
+          type="text"
+          placeholder="book title"
+          onChange={(e) => setTitle(e.target.value)}
+          value={title}
+        />
+        <input
+          type="text"
+          placeholder="category"
+          onChange={(e) => setCategory(e.target.value)}
+          value={category}
+        />
+        <button type="submit">submit</button>
+      </div>
```

Comment:
- [ ] The form responsible for adding a new book isn't properly styled(input fields and the horizontal line before the form), Kindly update your book component to match the Zeplin design.
Your styling
![image](https://user-images.githubusercontent.com/33767951/141313755-db1b121b-69bd-4abe-8892-f0fc1a8112c5.png)
Zeplin design
![image](https://user-images.githubusercontent.com/33767951/141313861-73635c84-7810-4c63-af79-54a6cbe8378b.png)

- [ ] Kindly, update the text of the form header to `ADD NEW BOOK`.
- [ ] Kindly, update the text of the submit button to `ADD BOOK`.
- [ ] The `Category` input should be a `select` field with are of the previously given categories.